test/parallel/test-global-descriptors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+
+const {
+  value,
+  configurable,
+  enumerable,
+  writable
+} = Object.getOwnPropertyDescriptor(global, 'global');
+
+assert.strictEqual(value, global, 'global should be global object');
```

Comment:
In that case `constructor` would be an own property on the function, and thus would shadow anything on `Function.prototype`, but I agree we don't need to worry about it :-)